workers/ohsome_quality_analyst/base/indicator.py

```diff
@@ -79,23 +80,35 @@ def __init__(
             svg=self._get_default_figure(),
         )
 
-    def as_feature(self) -> Feature:
+    def as_feature(self, flatten: bool = False) -> Feature:
         """Returns a GeoJSON Feature object.
 
         The properties of the Feature contains the attributes of the indicator.
         The geometry (and properties) of the input GeoJSON object is preserved.
+
+        Args:
+            flatten (bool): If true flatten the properties.
         """
         result = vars(self.result).copy()
-        # Prefix all keys of the dictionary
-        properties = {
-            "metadata.name": self.metadata.name,
-            "metadata.description": self.metadata.description,
-            "layer.name": self.layer.name,
-            "layer.description": self.layer.description,
-            **{"result." + str(key): val for key, val in result.items()},
-            **{"data." + str(key): val for key, val in self.data.items()},
-            **self.feature.properties,
-        }
+        if flatten:
+            unflattend = {
+                "metadata.name": self.metadata.name,
+                "metadata.description": self.metadata.description,
+                "layer.name": self.layer.name,
+                "layer.description": self.layer.description,
+                **{"result." + str(key): val for key, val in result.items()},
+                **{"data." + str(key): val for key, val in self.data.items()},
+                **self.feature.properties,
+            }
+            properties = flatten_dict(unflattend)
```

Comment:
I don't really see why you're using another properties json if you're using the flatten function. Wouldn't you miss layer introduced metadata and layer keys? Is there a disadvantage of just using the json of the `else:` part for both cases and just use:
```python
if flatten:
    properties = flatten_dict(properties)

Reply:
Adressed, so that properties is created once and flattend if flag is set.

Reply:
Need to check implementation agai. Hold on...

Reply:
Okay. Now its ready.